src/components/index.js

```diff
@@ -77,3 +77,7 @@ export { default as IconButton } from "./IconButton/IconButton";
 export { default as Flex } from "./Flex/Flex";
 export { default as MenuGridItem } from "./Menu/MenuGridItem/MenuGridItem";
 export { default as AvatarGroup } from "./AvatarGroup/AvatarGroup";
+export { default as Modal } from "./Modal/Modal";
```

Comment:
We need to add the components also to the published components file otherwise developers will not be able to access them.

Reply:
👍